components/templates/Book.tsx

```diff
@@ -147,7 +147,7 @@ export default function Book(props: Props) {
             <IconButton onClick={handleInfoClick}>
               <InfoOutlinedIcon />
             </IconButton>
-            {isInstructor && (
+            {((book && isBookEditable(book)) || book?.shared) && (
```

Comment:
`isInstructor` が取り除かれており、結果的に学習者がアクセスすると共有されているブックが表示されませんか。意図通りでしょうか。

Reply:
意図通りではないですね…考慮できておらず取り除きました。これで大丈夫でしょうか

```suggestion
            {isInstructor && ((book && isBookEditable(book)) || book?.shared) && (
```

Reply:
そうですね、学習者であれば編集権限を持たないので鉛筆ボタンが表示されない、という振る舞いを通常期待するのかなと考えます。

Reply:
> 学習者であれば編集権限を持たないので鉛筆ボタンが表示されない、という振る舞いを通常期待する

まったくもってそのとおりと思います